django/db/backends/ddl_references.py

```diff
@@ -103,6 +103,22 @@ def __str__(self):
         return self.create_index_name(self.table, self.columns, self.suffix)
 
 
+class IndexColumns(Columns):
+    def __init__(self, table, columns, quote_name, col_suffixes=(), opclasses=()):
+        self.opclasses = opclasses
+        super().__init__(table, columns, quote_name, col_suffixes)
+
+    def __str__(self):
+        def col_str(column, idx):
+            try:
+                col = self.quote_name(column) + self.col_suffixes[idx]
+            except IndexError:
+                col = self.quote_name(column)
+            return '{} {}'.format(col, self.opclasses[idx])
```

Comment:
Can we add a comment here that we ensure `self.opclasses` has the same length as `self.columns` in the `Index` class?